src/components/CardMovie.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <v-container>
+    <div class="title text-center text-h4 font-weight-bold green--text text--darken-3">{{title}}</div>
+    <div v-for="movie in movies" :key="movie.id" class="card-movie">
+      <img class="card-movie__img" :src="'https://image.tmdb.org/t/p/w500' + movie.poster_path" alt="poster">
+      <div class="card-movie__descr">
+        <div class="card-movie__descr-name">
+          <span>Name of movie: </span>
+          <span>{{ movie.original_title }}</span>
+        </div>
+        <div class="card-movie__descr-overview">
+          <span>Overview: </span>
+          <span>{{ movie.overview }}</span>
+        </div>
+      </div>
+        <v-btn class="card-movie__btn" :to="{ name: 'detailsAboutMovie', params: { id: movie.id }}" rounded>More</v-btn>
+    </div>
+  </v-container>
+</template>
```

Comment:
I've revived it several times. Should be deleted

Reply:
Deleted